src/librustdoc/html/static/css/themes/dark.css

```diff
@@ -3,6 +3,17 @@ body, #settings-menu #settings, #settings-menu #settings::before {
 	color: #ddd;
 }
 
+.setting-line .radio-line input {
+	border-color: #ddd;
+}
+.setting-line .radio-line input:checked {
+	box-shadow: inset 0 0 0 3px #353535;
+	background-color: #2196f3;
```

Comment:
What's this color? Can we reuse an existing color from the theme?

Reply:
It's the color from the "sliders" (the blue).

Reply:
I see that's set in settings.css:

```
input:checked + .slider {
    background-color: #2196F3;
}
```

But I think we always set colors in theme files, right? So I think that default blue color should get moved to light.css (even though it only applies to settings pages).

Reply:
Outch, that shouldn't have been. I'll move it into the theme files.